pages/docs/react/latest/styling.mdx

```diff
@@ -148,12 +149,27 @@ The most minimalistic approach is to create simple bindings to e.g. [`emotion`](
 This will give you straight-forward access to the `emotion` apis. Here's how you'd use them in your app code:
 
 ```res
-let app = <div
-  className={Emotion.css({
+let container = Emotion.css({
+  "color": "#fff",
+  "backgroundColor": "red"
+})
+
+let app = <div className={container} />
+```
+
+You can organize your module stylesheet in a React Native fashion using first-class modules!
```

Comment:
There are a few things that make this sentence too complex IMO:
- React Native -> We never mentioned it, nor do I think it should be mentioned to keep it lean
- first-class-modules -> I instantly thought about FCM, what you meant is a submodule, right?

```suggestion
You can also use submodules to organize your styles more easily:
```